code/datums/components/plumbing/automated_iv.dm

```diff
@@ -0,0 +1,26 @@
+/datum/component/plumbing/automated_iv
+	demand_connects = SOUTH
+	supply_connects = NORTH
+	var/obj/machinery/iv_drip/plumbing/myivdrip
```

Comment:
```suggestion
	///Temporary holder to store all the reagents from the iv drip before transferring it to the ducts
	var/datum/reagents/plumbing/holder
```
Don't store references like this when we can directly get it from the parent.

We will instead create our holder here